common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java

```diff
@@ -378,7 +383,7 @@ private static void log(final String tag,
                             final String message,
                             final Throwable throwable,
                             final boolean containsPII) {
-        if (logLevel.compareTo(sLogLevel) > 0 || (!sAllowPii && containsPII)) {
+        if (logLevel.compareTo(sLogLevel) > 0 || (!sAllowPii && containsPII)  || (sLogLevel == LogLevel.NO_LOG)) {
```

Comment:
>(sLogLevel == LogLevel.NO_LOG) [](http://example.com/codeflow?start=81&length=30)

Can be optimized by having this as first check

Reply:
done